Manipulator/ServicesManipulator.php

```diff
@@ -99,4 +145,24 @@ public function addResource($serviceId, $modelClass, $adminClass, $controllerNam
             ));
         }
     }
+
+    /**
+     * @param string $servicesContent
+     * @param string $serviceId
+     * @param string $adminClass
+     * @param string $modelClass
+     * @param string $controllerName
+     * @param string $managerType
+     *
+     * @return string
+     */
+    private function createServiceDefinitionXmlNode($servicesContent, $serviceId, $adminClass, $modelClass, $controllerName, $managerType)
+    {
+
+        $template = sprintf($this->xmlServiceDefinitionTemplate, $serviceId, $adminClass, $modelClass, $controllerName, $managerType, current(array_slice(explode('\\', $modelClass), -1)));
+
+        $content = str_replace('</services>', $template, $servicesContent);
```

Comment:
Good point… I though it was more a matter of having less code here, but if it messes up breaks and indention people are not going to be happy.